cpa/tester/policies/results.xml

```diff
@@ -1,4 +1,15 @@
-<testsuites name="root" tests="38" failures="0" errors="0" time="0">
+<testsuites name="root" tests="46" failures="0" errors="0" time="0">
```

Comment:
time will be populated in real output? (ELapsedMS in json today)

Reply:
NM! I Saw this is in the Result.go.  Sweet!

Reply:
You know it! But when testing against static data, time is the enemy 🙈